source/devices/J7_Family/linux/_Release_Specific_Kernel_Performance_Guide.rst

```diff
@@ -473,36 +473,35 @@ ALSA SoC Audio Driver
 
 
 .. csv-table:: Audio Capture
-    :header: "Sampling Rate (Hz)","j721e-idk-gw: Throughput (bits/sec)","j721e-idk-gw: CPU Load (%)","j721s2-evm: Throughput (bits/sec)","j721s2-evm: CPU Load (%)","j722s_evm-fs: Throughput (bits/sec)","j722s_evm-fs: CPU Load (%)","j784s4-evm: Throughput (bits/sec)","j784s4-evm: CPU Load (%)"
+    :header: "Sampling Rate (Hz)","j721e-idk-gw: Throughput (bits/sec)","j721e-idk-gw: CPU Load (%)","j721s2-evm: Throughput (bits/sec)","j721s2-evm: CPU Load (%)","j722s_evm-fs: Throughput (bits/sec)","j722s_evm-fs: CPU Load (%)","j784s4-evm: Throughput (bits/sec)","j784s4-evm: CPU Load (%)","j742s2-evm: Throughput (bits/sec)","j742s2-evm: CPU Load (%)"
 
-    "8000","","","","","255988.00","0.32","1023979.00","0.74"
-    "11025","352791.00","0.19","1023979.00","0.72","331809.00","0.36","1023988.00","0.27"
-    "16000","511990.00","0.33","1023988.00","1.21","511977.00","0.36","1023966.00","0.16"
-    "22050","705573.00","0.31","1023965.00","0.95","663631.00","0.45","1023982.00","0.21"
-    "24000","705581.00","0.35","1023982.00","0.93","663611.00","0.45","1023985.00","0.22"
-    "32000","1023977.00","0.22","1023984.00","1.04","1023926.00","0.54","1023966.00","0.15"
-    "44100","1411171.00","0.54","1417793.00","0.96","1327227.00","0.63","1417810.00","0.18"
-    "48000","1535969.00","0.75","1535955.00","0.97","1535909.00","0.66","1535967.00","0.14"
-    "88200","2822343.00","0.93","2835618.00","1.44","2654473.00","1.00","2835634.00","0.32"
-    "96000","3071938.00","0.50","3071926.00","1.52","3071829.00","1.05","3071939.00","0.20"
+    "8000","","","","","255988.00","0.32","1023979.00","0.74","1023983.00","0.31"
+    "11025","352791.00","0.19","1023979.00","0.72","331809.00","0.36","1023988.00","0.27","1023990.00","0.24"
+    "16000","511990.00","0.33","1023988.00","1.21","511977.00","0.36","1023966.00","0.16","1023972.00","0.31"
+    "22050","705573.00","0.31","1023965.00","0.95","663631.00","0.45","1023982.00","0.21","102398.00","0.44"
+    "24000","705581.00","0.35","1023982.00","0.93","663611.00","0.45","1023985.00","0.22","1023988.00","0.39"
+    "32000","1023977.00","0.22","1023984.00","1.04","1023926.00","0.54","1023966.00","0.15","1023967.00","0.31"
+    "44100","1411171.00","0.54","1417793.00","0.96","1327227.00","0.63","1417810.00","0.18","1417812.00","0.43"
+    "48000","1535969.00","0.75","1535955.00","0.97","1535909.00","0.66","1535967.00","0.14","1535969.00","0.29"
+    "88200","2822343.00","0.93","2835618.00","1.44","2654473.00","1.00","2835634.00","0.32","2835638.00","0.67"
+    "96000","3071938.00","0.50","3071926.00","1.52","3071829.00","1.05","3071939.00","0.20","3071945.00".,"0.40"
```

Comment:
Without completely parsing this I assume this is the issue the linter is upset about.

```suggestion
    "96000","3071938.00","0.50","3071926.00","1.52","3071829.00","1.05","3071939.00","0.20","3071945.00","0.40"
```